tests/api/test_patch.py

```diff
@@ -0,0 +1,74 @@
+import pickle
+from lightly.api import api_workflow_client
+from lightly.api import ApiWorkflowClient
+from pytest_mock import MockerFixture
+from lightly.openapi_generated.swagger_client import ApiClient, Configuration
+from lightly.openapi_generated.swagger_client.rest import RESTClientObject
+
+
+def test_make_swagger_generated_classes_picklable__api_workflow_client(
+    mocker: MockerFixture,
+) -> None:
+    mocker.patch.object(api_workflow_client, "is_compatible_version", return_value=True)
+    client = ApiWorkflowClient(token="123")
+    client._dataset_id = "my-dataset-id"
+    new_client = pickle.loads(pickle.dumps(client))
+
+    assert set(client.__dict__.keys()) == set(new_client.__dict__.keys())
+    assert all(
+        type(client.__dict__[key] == new_client.__dict__[key])
```

Comment:
Why `type()` ? Or should it be around each separately?

Reply:
Great catch, thanks! That should indeed be individual types. Fixed it.